Gruntfile.js

```diff
@@ -85,8 +85,51 @@ module.exports = function(grunt) {
             }
         },
         clean: {
+            coverage: ['coverage/', 'coverage.lcov', '.nyc_output/'],
             dist: ['.dist/'],
+            packages: ['Consus-Client-*/'],
             test: ['.test/']
+        },
+        electron: {
+            win32Build: {
+                options: {
+                    name: 'Consus-Client',
+                    dir: '.',
+                    out: '.',
+                    version: '1.4.2',
+                    platform: 'win32',
+                    arch: 'x64',
+                    'no-prune': true,
+                    overwrite: true,
+                    asar: true,
+                    icon: 'consus-logo.ico' //This isn't working?
+                }
+            },
+            // macosBuild: {
```

Comment:
I don't think we should have this left out, since I can build for macOS on Linux (which Travis is also using). You Windows users will just have to remember to use `grunt build && grunt electron:win32Build`.

Just because we don't *need* to package for macOS, in my opinion, doesn't mean we should not support it since it is so easy to do so.